contracts/mocks/ExampleERC20.sol

```diff
@@ -13,12 +13,13 @@ import {
     ERC20Burnable,
     ERC20
 } from "@openzeppelin/contracts@5.0.2/token/ERC20/extensions/ERC20Burnable.sol";
+import {IERC20Mintable} from "../staking/interfaces/IERC20Mintable.sol";
 
-contract ExampleERC20 is ERC20Burnable {
+contract ExampleERC20 is ERC20Burnable, IERC20Mintable {
     string private constant _TOKEN_NAME = "Mock Token";
     string private constant _TOKEN_SYMBOL = "EXMP";
 
-    uint256 private constant _MAX_MINT = 1e16;
+    uint256 private constant _MAX_MINT = 1e19;
 
     constructor() ERC20(_TOKEN_NAME, _TOKEN_SYMBOL) {
```

Comment:
thinking if this example ERC20 should only give access to mint to an ownable address. Even for an example seems like a pretty common check that would exist.

Reply:
I'm not quite sure what you mean